frontend/src/metabase/core/components/AccordionList/AccordionList.jsx

```diff
@@ -181,6 +180,12 @@ export default class AccordionList extends Component {
     }
   }
 
+  _getRootContainer() {
+    return this.isVirtualized()
+      ? this._list?.Grid?._scrollingContainer
+      : this.listRootRef.current;
+  }
```

Comment:
`findDOMNode` was resolving to the scrolling container when virtualized, otherwise it resolves to the accordion root.